tests/crescent/plugins/test_plugins.py

```diff
@@ -92,8 +92,8 @@ def test_load_folder(self, caplog: LogCaptureFixture):
             plugin as nested_plugin,
         )
 
-        # No messages should be logged if plugins are loaded successfully.
-        assert len(caplog.messages) == 0
+        # Only the -OO warning should be logged if plugins are loaded successfully.
+        assert len(caplog.messages) == 1
```

Comment:
I feel like it would be better to check the message content exactly.

Reply:
I don't want to test for something not being included because if the log message is changed the test would have to be changed but it wouldn't fail.